apps/rpc-server/src/index.ts

```diff
@@ -229,6 +210,59 @@ function determinePort(portEnv?: string) {
     return defaultPort;
 }
 
+function versionListener({ rpcServer }: DPapi.Versions) {
+    const cmp = Utils.versionCompare(rpcServer, Version);
+    if (Res.isErr(cmp)) {
+        log.error('error comparing versions: %s', cmp.error);
+        return;
+    }
+    const logErr = () => {
+        const errMessage = [
+            `*** RPCServer[v${Version}] outdated and will not work -`,
+            `please update to latest version v${rpcServer}.`,
+            'Visit https://degen.rpch.net for detail! ***',
+        ].join(' ');
+        const errDeco = Array.from({ length: errMessage.length }, () => '*').join('');
+        log.error(`!!! ${errDeco} !!!`);
+        log.error(`!!! ${errMessage} !!!`);
```

Comment:
Would `log.error('\x1b[33m%s\x1b[0m', `!!! ${errMessage} !!!`);` work here as with console.log?

Reply:
Unfortunately not